.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,80 @@
+name: 🪲 Bug report
+description: Create a report to help us improve
+
+body:
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      options:
+      - label: I have written a descriptive issue title
+        required: true
+      - label: | 
+          I have searched existing issues to ensure the bug has not already been reported
+        required: true
+
+  - type: input
+    id: mongoose-version
+    attributes:
+      label: Mongoose version
+      placeholder: 6.x.x
+    validations:
+      required: true
+
+  - type: input
+    id: node-version
+    attributes:
+      label: Node.js version
+      placeholder: 14.x
+    validations:
+      required: true
+  
+  - type: input
+    id: mongo-version
+    attributes:
+      label: MongoDB version
+      placeholder: 5.x
+    validations:
+      required: true
+
+  - type: dropdown
```

Comment:
Let's get rid of the OS input, this is rarely significant

Reply:
done